src/config.h

```diff
@@ -1,5 +1,7 @@
-/* src/config.h.  Generated from config.h.in by configure.  */
+#include <xapian.h>
+
+/* Xapian::RangeProcesor (and subclasses) was introduced in 1.3.6. Therefore, with Xapian versions before 1.3.6, RXapian uses Xapian::ValueRangeProcessor (and subclasses) for supporting range queries. */
 #if XAPIAN_MAJOR_VERSION >= 2 || XAPIAN_MINOR_VERSION >=3 || (XAPIAN_MINOR_VERSION == 3 && XAPIAN_REVISION >= 6)
-#define XAPIAN_RANGE_PROCESSOR_SUPPORT
+#define XAPIAN_RANGE_PROCESSOR_SUPPORT && XAPIAN_OP_MAX_SUPPORT
```

Comment:
Thanks. Added a separate test for it.

On Thu, Aug 11, 2016 at 9:05 PM, James Aylett <notifications@github.com>
wrote:

> In src/config.h
> <https://github.com/amandaJayanetti/RXapian/pull/25#discussion_r74493178>:
>
> >  #if XAPIAN_MAJOR_VERSION >= 2 || XAPIAN_MINOR_VERSION >=3 || (XAPIAN_MINOR_VERSION == 3 && XAPIAN_REVISION >= 6)
> > -#define XAPIAN_RANGE_PROCESSOR_SUPPORT
> > +#define XAPIAN_RANGE_PROCESSOR_SUPPORT && XAPIAN_OP_MAX_SUPPORT
>
> 1.3.2, I believe. (It's hard to figure out.)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/amandaJayanetti/RXapian/pull/25/files/72fb99ec5690760ac99cc99f918eccadd124af78..9a9b947386dd1fd196aaa25f4d4a8952a7c35b81#r74493178>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AH_jNs8CBd-Z2iOpel9tjXEOilQwcPBgks5qe4BsgaJpZM4Jf4Xo>
> .
>